src/sidebar/media-embedder.js

```diff
@@ -49,7 +49,7 @@ function parseTimeString(timeValue) {
   // match[2] - Unit (e.g. 'h','m','s', or empty)
   while ((match = timePattern.exec(timeValue)) !== null) {
     if (match[2]) {
-      seconds += match[1] * multipliers[match[2]];
+      seconds += Number(match[1]) * multipliers[match[2]];
```

Comment:
This is just making an implicit string => number conversion explicit.

Reply:
Good.